src/main/java/tech/andreagreco/Message.java

```diff
@@ -0,0 +1,16 @@
+package tech.andreagreco;
+
+public class Message {
+
+    String address;
+    String timestamp;
+    String type;
+
+    public Message() {
+        super();
+    }
+
+    public String getAdress() {
```

Comment:
Could you please correct this typo?
```suggestion
    public String getAddress() {
```